### ISSUE TITLE
Post Feedback (ISSUES)  even over Twitter.  

### ISSUE THREAD

**Issue by skyerjoe** [corona-warn-app/cwa-wishlist#259]
## Current Implementation

Post Issues over Twitter --> Github
## Suggested Enhancement

for ex. use Zapier. https://zapier.com/apps/github/integrations/twitter
## Expected Benefits

The normal User has no time to challenge himself with Github. 

Best regards

**Comment by heinezen** [comment]
Hey @skyerjoe ,

Thanks you for the suggestion. We have created a ticket in our internal Jira for the request (EXPOSUREAPP-3644) and forwarded the feedback to the development team. It will now be discussed how and if the feature is implemented. Updates on the progress or status are posted here on Github.

Regards,
CH

----
Corona-Warn-App Open Source Team

**Comment by achisto** [comment]
I see another benefit with a suggestion like that; currently the official CWA Twitter account is rather inactive (last tweet from 04. August) and not used at all to have a dialogue although there are dozens of tweets mentioning the account, asking questions and reporting unexpected behaviour. In contrast to GitHub, where the user base tends to be a bit more tech-savvy, on Twitter you find all kinds of people which is a lot closer to the kind of everyday user. Starting a conversation with everyday users instead of just "techies" could lead to interesting suggestions and uncover problems one wouldn't have seen otherwise.

**Comment by Ein-Tim** [comment]
@achisto 

Regarding [your comment](https://github.com/corona-warn-app/cwa-wishlist/issues/259#issuecomment-724154437), I opened #364.

**Comment by Ein-Tim** [comment]
Little update here, I'm trying my best to monitor all Tweets reg. the Corona-Warn-App. If I see somebody tweeting that he/she has issues with the CWA, I investigate a bit and report the issue here on GitHub. 

For example, these issues are based on information from Tweets: https://github.com/corona-warn-app/cwa-app-android/issues/3561, https://github.com/corona-warn-app/cwa-app-android/issues/3559 & https://github.com/corona-warn-app/cwa-app-android/issues/3558.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Do the Twitter users know about the [@coronawarnapp Twitter Channel](https://twitter.com/coronawarnapp) announced yesterday in https://www.coronawarn.app/en/blog/2021-06-29-cwa-twitter-account/?

I suggested to RKI two weeks ago that they add [@coronawarnapp](https://twitter.com/coronawarnapp) to their e-mail signature from the [CoronaWarnApp@rki.de](mailto:CoronaWarnApp@rki.de) e-mail account and they have implemented that now. Previously they only listed [@rki_de](https://twitter.com/rki_de).

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Yes, most of the users know about the account and also mention it ([@coronawarnapp](https://twitter.com/coronawarnapp)), but the maintainers of the account do not respond to any Tweets from the community.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> ... the maintainers of the account do not respond to any Tweets from the community.

So it looks like Twitter is not viewed as a support channel, although I saw some generic replies today in https://twitter.com/coronawarnapp/status/1410175936704679937?s=20.

The [announcement](https://www.coronawarn.app/en/blog/2021-06-29-cwa-twitter-account/) said: "We look forward to discussions, suggestions for improvement and experiences in using the CWA."
